### PR TITLE
Add support for multiple profiles with credentials file

### PR DESCRIPTION
**Pull request by GeertWille**
*Issue #, if available:* #112

*Description of changes:*
1. Added prettier to format code to be consistent
2. Added multiple profile support
---

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws-actions/configure-aws-credentials/blob/master/CONTRIBUTING.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: dist/cleanup/index.js [hunk 1]
```diff
@@ -23,13 +24,14 @@ async function cleanup() {
...
-    core.exportVariable('AWS_REGION', '');
-  }
-  catch (error) {
+    core.exportVariable("AWS_ACCESS_KEY_ID", "");
+    core.exportVariable("AWS_SECRET_ACCESS_KEY", "");
+    core.exportVariable("AWS_SESSION_TOKEN", "");
+    core.exportVariable("AWS_DEFAULT_REGION", "");
+    core.exportVariable("AWS_REGION", "");
+
+    fs.rmSync(`${process.env.HOME}/.aws`, { force: true, recursive: true });
```
Comment (k2589) [inline dist/cleanup/index.js:33]: @GeertWille Please recheck this line/imports
Looks like something went wrong, getting this error:
```
Post job cleanup.
Error: fs.rmSync is not a function
```

**Comment by sarink** [comment]
This is much needed!

**Comment by kidpollo** [comment]
Did this get closed in favor of something else?

**Comment by elreydetoda** [comment]
It looks like it got accidentally closed when they migrated from using master as the default to using main (when @kellertk deleted `aws-actions:master` before your comment).

**Comment by act-mreeves** [comment]
@GeertWille could you reopen this on main and perhaps @kellertk can move this fwd or at least give feedback?

**Pull request closed without merging**